From 43f9bd5f5e9cffec32e85ea0bb6aa41147282546 Mon Sep 17 00:00:00 2001
From: Robby Cai <r63905@freescale.com>
Date: Fri, 28 Aug 2015 15:31:07 +0800
Subject: [PATCH] MLK-11438-6 ARM: dts: enable epdc

enable epdc since lpsr mode is supported
disabled fec2 due to pin conflict, and there's another
dts file *-enet.dts for fec2.

Signed-off-by: Robby Cai <r63905@freescale.com>
---
 arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
index 4072485..99d84ced 100644
--- a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
@@ -195,7 +195,7 @@
         V3P3-supply = <&V3P3_reg>;
         VCOM-supply = <&VCOM_reg>;
         DISPLAY-supply = <&DISPLAY_reg>;
-        status = "disabled";
+        status = "okay";
 };
 
 &epxp {
@@ -243,7 +243,7 @@
 	phy-mode = "rgmii";
 	phy-handle = <&ethphy0>;
 	fsl,magic-packet;
-	status = "okay";
+	status = "disabled";
 };
 
 &flexcan1 {
-- 
1.8.0

